src/index.ts

```diff
@@ -1,3 +1,7 @@
+import 'regenerator-runtime/runtime'
```

Comment:
I don't see this dep in the package.json

Reply:
Oops, fixed.